org.musicbrainz.Picard.json

```diff
@@ -1,56 +1,152 @@
 {
-    "id": "org.musicbrainz.Picard",
+    "app-id": "org.musicbrainz.Picard",
     "runtime": "org.freedesktop.Platform",
     "runtime-version": "1.6",
     "sdk": "org.freedesktop.Sdk",
     "command": "picard",
     "rename-desktop-file": "picard.desktop",
     "rename-icon": "picard",
     "finish-args": [
+        /* X11 + XShm access */
         "--share=ipc",
         "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* OpenGL access */
+        "--device=dri",
+        /* Needs to talk to the network: */
         "--share=network",
+        /* GConf */
+        "--talk-name=org.gnome.GConf",
         "--filesystem=home",
-        "--filesystem=xdg-music",
-        "--talk-name=org.gnome.GConf"
+        "--filesystem=xdg-music"
     ],
     "modules": [
-        "shared-modules/qt4/qt4-4.8.7-minimal.json",
         {
-            "name": "sip",
+            "name": "qt5-qtbase",
```

Comment:
I'm currently talking with the maintainers of the KDE Sdk, to see if we can add PyQt to it, so we don't have to bundle it in Picard.

In any case, bundling Qt5 shouldn't be necessary, as they won't break the Qt ABI in a given SDK branch… unless PyQt uses some private APIs for which Qt doesn't guarantee the ABI.

Reply:
We can maybe create an extension ourselves (like https://github.com/flathub/org.freedesktop.Sdk.Extension.rust-stable for example) ?